incydr/cli/cmds/alerts.py

```diff
@@ -71,10 +76,10 @@ def search(
     if advanced_query:
         query = AlertQuery.parse_raw(advanced_query)
     else:
-        if not any([start, on]):
+        if not any([start, on, end]):
```

Comment:
Allowing only --end would be useful when bulk closing alerts older than <date>.

Reply:
True, does alerts work like the file events APIs where it will default to the oldest possible start date?

Reply:
If a query only has an end param and no start, it'll get all older alerts that exist, yeah. I don't know if alerts age out and get purged like file events do. My initial guess is no, since the scale of alerts is so much smaller than events it's not a cost issue for keeping them around.